app/uk/gov/hmrc/brm/models/response/Child.scala

```diff
@@ -18,19 +18,7 @@ package uk.gov.hmrc.brm.models.response
 
 import org.joda.time.LocalDate
 
-case class Child(
-                  birthReferenceNumber: Int,
-                  private val _forenames: String,
-                  private val _lastName: String,
-                  dateOfBirth: Option[LocalDate]
-                ) {
-
-  import uk.gov.hmrc.brm.services.parser.NameParser._
-
-  def forenames : String = _forenames.names.listToString
-
-  def lastName : String = _lastName.names.listToString
```

Comment:
Is there a reason these were deleted? It might mess with expected format of the names.

Reply:
The code behaves the same way - it was written in a dreadful way 
_forenames is a string, names was an implicit function to turn it into a list, then listToString turned it back into a string...

I simplified the code without changing the tests so behaviour shouldn't have changed... 👀